portfolio.html

```diff
@@ -16,84 +23,92 @@
         <h1>My Recent Works</h1>
+        <div>
+          <hr id="set1-solid" />
+        </div>
       </div>
       <div>
-        <hr class="solid" />
+        <hr class="mob-solid" />
       </div>
-      <div>
-        <img
-          class="set-image"
-          src="/images/ImgPlaceholder.png"
-          alt="works image"
-        />
-      </div>
-      <div id="set2">
-        <h2>Multi-Post Stories</h2>
-        <p>
-          A daily selection of privately personalized reads; no accounts or
-          sign-ups required. has been the industry's standard dummy text ever
-          since the 1500s, when an unknown printer took a standard dummy text.
-        </p>
-      </div>
-      <div id="lang">
-        <ul>
-          <li><img src="/images/Icons/languages/css.png" alt="csslang" /></li>
-          <li><img src="/images/Icons/languages/html.png" alt="htmllang" /></li>
-          <li>
-            <img src="/images/Icons/languages/bootstrap.png" alt="bootlang" />
-          </li>
-          <li><img src="/images/Icons/languages/ruby.png" alt="rubylang" /></li>
-        </ul>
-      </div>
-      <div class="project-button">
-        <a href="projectlink"
-          ><img
-            src="/images/Icons/Button/Enable.png"
-            alt="button
+      <div id="first-post">
+        <div>
+          <img
+            class="set-image"
+            src="images/ImgPlaceholder.png"
+            alt="works image"
+          />
+          <img
+            class="set-image-desk"
+            src="images/desktop/mpsdesktop.png"
+            alt="works image"
+          />
+        </div>
+        <div id="first-post-text">
+          <div id="set2">
+            <h2>Multi-Post Stories</h2>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard dummy text
+              ever since the 1500s, when an unknown printer took a standard
+              dummy text.
+            </p>
+          </div>
+          <div id="lang">
+            <ul>
+              <li>
+                <img src="images/Icons/languages/css.png" alt="csslang" />
+              </li>
+              <li>
+                <img src="images/Icons/languages/html.png" alt="htmllang" />
+              </li>
+              <li>
+                <img
+                  src="images/Icons/languages/bootstrap.png"
+                  alt="bootlang"
+                />
+              </li>
+              <li>
+                <img src="images/Icons/languages/ruby.png" alt="rubylang" />
+              </li>
+            </ul>
+          </div>
+          <div class="project-button">
+            <a href="projectlink"
+              ><img
+                src="images/Icons/Button/Enable.png"
+                alt="button
         icon"
-        /></a>
+            /></a>
+          </div>
+        </div>
       </div>
     </section>
     <section class="project-works">
```

Comment:
- [ ] **GOOD JOB SO FAR 👍👏🔥**. One one is missing in the project section that there should be 6 projects instead of one. Please fix this issue.